src/commands/shared/ssh.ts

```diff
@@ -143,12 +142,28 @@ export const provisionRequest = async (
     throw "Public key mismatch. Please revoke the request and try again.";
   }
 
+  return { provisionedRequest, publicKey, privateKey };
+};
+
+export const prepareRequest = async (
+  authn: Authn,
+  args: yargs.ArgumentsCamelCase<BaseSshCommandArgs>,
+  destination: string
+) => {
+  const result = await provisionRequest(authn, args, destination);
+  if (!result) {
+    throw "Server did not return a request id. Please contact support@p0.dev for assistance.";
```

Comment:
Would users know what this means? What if we used something more generic like: `Server failed to process your request. Please contact ...`

Reply:
Good point. Maybe we should have an error code? It is useful for us to know the underlying issue when they contact us.
In the backend and frontend we have an error boundary that prints an error ID they can share with us. Filed ENG-2498